pinot-common/src/main/java/org/apache/pinot/common/utils/DatabaseUtils.java

```diff
@@ -27,6 +27,8 @@
 import org.apache.pinot.spi.exception.DatabaseConflictException;
 import org.apache.pinot.spi.utils.CommonConstants;
 
+import static org.apache.pinot.spi.utils.CommonConstants.DATABASE;
```

Comment:
Is static imports allowed? I remember checkstyle failing for me. I see that github actions passed. Surprised.

Reply:
I think `import static` is less recommended since sometime ago, but they are not cleaned up from the repo yet. So better avoid it when possible.